pkg/cmd/pulumi/org_search.go

```diff
@@ -129,6 +192,14 @@ func newSearchCmd() *cobra.Command {
 			"Must be formatted like: -q key1=value1 -q key2=value2. "+
 			"Alternately, each parameter provided here can be in raw Pulumi query syntax form.",
 	)
+	cmd.PersistentFlags().VarP(
+		&scmd.outputFormat, "output", "o",
+		"Output format. Supported formats are 'table', 'json', 'csv', and 'yaml'.",
+	)
```

Comment:
Not blocking for this PR, but as a follow-up, it might be nice to add support for `--json`, which, if selected would choose JSON as the output format (and would error if you specified `--output` with some other value). This would align with many other CLI commands that support `--json`.

Reply:
Ah! Yes, I like that as well. Would you mind opening an issue on `pulumi/pulumi.ai`? Once this is ready to release, I'd love to come back and address these usability issues as well.